tools/check_volume_levels.m

```diff
@@ -251,3 +279,29 @@ function plot_levels(meas, tc, lm)
 		y(:,j) = filter(b, a, x(:,j));
 	end
 end
+
+% This function is copy of
+% sof/tools/test/audio/test_utils/multitone.m
+function x = multitone( fs, f, amp, tlength )
+	n = round(fs*tlength);
+	t = (0:n-1)/fs;
+	nf = length(f);
+	if nf > 1
+		ph = rand(nf, 1)*2*pi;
+	else
+		ph = 0;
+	end
+
+	x = zeros(n, 1);
+	for i=1:length(f)
+		x = x + amp(i)*sin(2*pi*f(i)*t+ph(i))';
```

Comment:
can you please a space for the equation x = x + amp(i) * sin(2 * pi *f(i) * t + ph(i))';